code/modules/surgery/organs/stomach/_stomach.dm

```diff
@@ -205,6 +217,18 @@
 /obj/item/organ/stomach/get_availability(datum/species/owner_species)
 	return !(NOSTOMACH in owner_species.inherent_traits)
 
+/obj/item/organ/stomach/proc/handle_contents(mob/living/carbon/human/mule, delta_time)
+	var/inedibles = 0
```

Comment:
```suggestion
	var/inedibles = 0
	if(is_clown_job(mule.mind?.assigned_role)))
		return
```
through years of classical stage training, clowns have learned how to handle eating inedible things

Reply:
You can't deep fry the disk. Although you could if eating it wouldn't destroy it.